index.html

```diff
@@ -102,6 +102,35 @@ <h3 class="subHeading">
         </div>
       </div>
     </section>
+    <section class="section2 sectionWrap backgroundImg" id="section2">
+      <div class="inner">
+        <div class="headWrap">
+          <h2 class="mainHead">PIX TECHの思い</h2>
+          <p class="subHead">プログラミングの仕事を楽しむために</p> 
+        </div>
+        <div class="contentsWrap">
+          <p class="text">
+            PIX TECH(ピクステック)のカリキュラムは、大手企業での業務を経験した現役のプログラマーが教える、現場での仕事の進め方に乗っ取り、未経験から現場で活躍できるプログラマーを目指し、実際に現場に入って仕事ができるようになるところまでをゴールとしたカリキュラムです。
+            <br>
```

Comment:
```suggestion
            PIX TECH(ピクステック)のカリキュラムは、大手企業での業務を経験した現役のプログラマーが教える、現場での仕事の進め方に乗っ取り、未経験から現場で活躍できるプログラマーを目指し、実際に現場に入って仕事ができるようになるところまでをゴールとしたカリキュラムです。<br>
```
brタグの後に改行しないと、末尾に半角スペースができてしまうので、気を付けましょう。

Reply:
> brタグの後に改行しないと、末尾に半角スペースができてしまうので、気を付けましょう。

修正しました。
371082df720c25dd0377c1928c4047a5335740fa